xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -172,6 +173,8 @@
       ScheduledExecutorService timeService,
       BackoffPolicy.Provider backoffPolicyProvider,
       Supplier<Stopwatch> stopwatchSupplier) {
+    logId = InternalLogId.allocate("xds-client", null);
```

Comment:
You can pass `targetName` to the 2nd arg of `allocate()`

Reply:
Done.